src/components/moviesDescription.js

```diff
@@ -0,0 +1,59 @@
+import axios from 'axios';
+import { Component } from 'react';
+
+import './movieDescription.css';
+
+
+class MoviesDscription extends Component {
+    _isMounted = false;
+    constructor(props) {
+        super(props);
+     
+        this.state = {
+          movies: [],
+        };
+      }
+  
+      componentDidMount(){
+        this._isMounted = true;
+        const moviesId = this.props.match.params.movieId;
+        axios.get(`
+        https://api.themoviedb.org/3/movie/${moviesId}?api_key=ed4f70d18fa8cbc03fb59ce04847e1bc`)
```

Comment:
same API key is being used in multiple components, store it in a different file as constant and import where required